### PR TITLE
[tests-only][full-ci]Test for getting group with utf8 characters in name

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
This PR adds the test for getting details of group having UTF-8 characters in group name.

## Related Issue

Related to issue: https://github.com/owncloud/ocis/issues/5567

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/TestHelpers/GraphHelper.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ private static function getRequestHeaders(): array {
 	 * @return string
 	 */
 	public static function getUUIDv4Regex(): string {
-		return '[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}';
+		return '[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}';
 	}
```
Comment (SwikritiT) [inline tests/TestHelpers/GraphHelper.php:35]: I see these changes in 3 of your PR's, if the PR's are branched out from one another and there's dependency better to keep the PR in draft until the main PR is merged.

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

File: tests/acceptance/features/apiGraph/getGroup.feature [hunk 2]
```diff
@@ -132,3 +132,29 @@ Feature: get groups and their members
       }
     }
     """
+
+
+  Scenario: Get details of group with UTF-8 characters name
+    Given group "$x<=>[y*z^2+1]!" has been created
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/getGroup.feature:138]: IMO, here UTF-8 really means the Unicode characters
```suggestion
    Given group "<some-unicode-chars>" has been created
```
CC @individual-it @ScharfViktor

**Review by SwikritiT — APPROVED** [review]
LGTM

File: tests/acceptance/features/apiGraph/getGroup.feature [hunk 3]
```diff
@@ -132,3 +132,35 @@ Feature: get groups and their members
...
+        },
+        "id": {
+          "type": "string",
+          "pattern": "^%group_id_pattern%$"
+        }
+      }
+    }
+    """
+    Examples:
+      | some-unicode-chars |
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/getGroup.feature:162]:
```suggestion
      | group |
```

**Review by saw-jan — COMMENTED** [review]
LGTM

**Pull request merged**